docs/source/tutorials/single_npu_qwen3vl.md

```diff
@@ -0,0 +1,186 @@
+# Single NPU (Qwen3-VL 4B)
```

Comment:
https://github.com/vllm-project/vllm/blob/main/examples/offline_inference/vision_language.py#L1459

OK, Although Qwen3 VL 4B has not been released yet, the vLLM upstream has already include this model name, so I think uploading the document is OK. If it is released, please do a post verification.

Reply:
model is here https://huggingface.co/Qwen/Qwen3-VL-235B-A22B-Instruct . the problem now is it's rely on a new release of transformers.